packages/atlas/src/components/NotificationTile/NotificationTile.tsx

```diff
@@ -0,0 +1,91 @@
+import { differenceInCalendarYears, differenceInDays, format } from 'date-fns'
+import React, { useMemo } from 'react'
+
+import { Avatar } from '@/components/Avatar'
+import { Text } from '@/components/Text'
+import { Checkbox } from '@/components/_inputs/Checkbox'
+import { SkeletonLoader } from '@/components/_loaders/SkeletonLoader'
+import { formatDateAgo } from '@/utils/time'
+
+import { AvatarWrapper, Content, Title, Wrapper } from './NotificationTile.styles'
+
+export type NotificationProps = {
+  id: string
+  author: string
+  text: string
+  date: Date
+  avatarUrl: string
+  videoTitle: string
+  loading?: boolean
+  onSelect?: (id: string) => void
+  read?: boolean
+  selected?: boolean
+  variant?: 'default' | 'compact'
+  className?: string
+}
+
+export const NotificationTile: React.FC<NotificationProps> = ({
+  id,
+  author,
+  text,
+  date,
+  avatarUrl,
+  videoTitle,
+  loading,
+  onSelect,
+  read,
+  selected = false,
+  variant = 'default',
+  className,
+}) => {
+  const titleVariant = variant === 'default' ? 'h300' : 't200-strong'
+  const avatarSkeletonSize = variant === 'default' ? 40 : 32
+  const formattedDate = useMemo(() => {
+    const differenceDays = differenceInDays(new Date(), date)
+    const differenceYears = differenceInCalendarYears(new Date(), date)
+    if (differenceYears >= 1) {
+      return format(date, 'dd LLL yyyy')
+    }
+    if (differenceDays > 3) {
+      return format(date, 'LLL d')
+    }
+    return formatDateAgo(date)
+  }, [date])
+
+  return (
+    <Wrapper read={read} selected={selected} loading={loading} className={className} variant={variant}>
+      {!loading ? (
+        variant === 'default' && <Checkbox onChange={() => onSelect?.(id)} value={selected} />
+      ) : (
+        <SkeletonLoader width={16} height={16} />
```

Comment:
You also need to add border-radius.
yours:
![image](https://user-images.githubusercontent.com/51168865/155971943-e62de35a-b960-45e9-8371-81586f95faf9.png)
design: 
![image](https://user-images.githubusercontent.com/51168865/155972319-ffaf965b-3cf5-4eda-96b3-decc79b4e094.png)
I think it should be 2px